test/routes/root.test.ts

```diff
@@ -3,9 +3,36 @@ import { build } from '../helper';
 
 test('default root route', async (t) => {
   const app = await build(t);
-
+  //root
   const res = await app.inject({
     url: '/',
   });
   t.same(JSON.parse(res.payload), { root: true });
-});
+
+  //fetcher1
```

Comment:
Put the tests in a separate directory, also for each fetcher we must have a separate test.

Reply:
done.